episodes/05-infrastructure.md

```diff
@@ -0,0 +1,196 @@
+Team](https://carpentries.org/core-team-projects/#curriculum-team), with the help of [Maxim
+Belkin](https://github.com/maxim-belkin).
+
+
+::::::::::::::::::::::::::::::::::::::::::::::::::
+
+### Lesson homepage
+
+The lesson homepage is built from the `index.md` file and provides an overview of the lesson,
+including any prerequisites, an introduction to the dataset used, and any other information learners will need for the lesson.
+
+The Instructor view of the lesson homepage also includes a schedule showing the individual episodes included in the lesson,
+and the time allotted for each. The schedule is automatically created based on information present in
+the episode files.
+
+### Episodes
+
+The majority of a lessons content is in its episode files. Episode files are stored in the
+`_episodes/` folder within your lesson repo (or in `_episodes_rmd/` for lessons written in R).
+Episode files are written in *Markdown* or *RMarkdown*. The essential elements of an episode
+are mentioned below. For complete information about how to contrust each element
+of an episode file, please see [the documentation](https://carpentries.github.io/sandpaper-docs/episodes.html). 
+
+#### Episode headers
+
+Each episode starts with a YAML header that looks like this:
+
+```
+---
+title: "What is the shell?"
+teaching: 5
+exercises: 0
+---
+```
+
+The information stored in the YAML header is used by the lesson infrastructure to populate the schedule.
+
+
+#### Questions, objectives, and keypoints
+
+Each episode must contain one of each of the following types of callouts: 
+
+- Questions: Located at the top of the episode file, these are plain language questions that show learners what to expect to gain from the episode.
+- Objectives: Located at the top of the episode file, objectives are specific goals that learners should be able to accomplish by the end of the episode.
+- Keypoints: Located at the end of the episode file, key points are statements of the major take-aways learners should remember from each episode. 
+
+Keypoints are automatically gathered from each episode and collected on the "Key Points" page of each lesson. Questions and Objectives are represented
+in an "Overview" box at the top of each rendered episode page.
+
+#### Callouts
+
+While not essential, most episodes will contain one or more callouts. Callouts can be
+of different types, with the most common being challenges (or exercises) and Instructor notes. The general callout syntax uses 
+fenced divs, as shown below. 
+
+```
+::::::::::::::::::::::::::::::::::::::: callout
+
+Some text here.
+
+:::::::::::::::::::::::::::::::::::::::
+
+```
+
+Callouts with the tag "challenge" will render as exercises:
+
+```
+::::::::::::::::::::::::::::::::::::::: challenge
+
+This is a challenge.
+
+:::::::::::::::::::::::::::::::::::::::
+
+```
+
+::::::::::::::::::::::::::::::::::::::: challenge
+
+This is a challenge.
+
+:::::::::::::::::::::::::::::::::::::::
```

Comment:
```suggestion
Callouts with the tag "challenge" will render as exercises:

````
::::::::::::::::::::::::::::::::::::::: challenge

This is a challenge.

:::::::::::::::::::::::::::::::::::::::

````

::::::::::::::::::::::::::::::::::::::: challenge

This is a challenge.

:::::::::::::::::::::::::::::::::::::::

Another block with the tag "solution" can be nested inside a challenge to provide the solution to an exercise:

````
::::::::::::::::::::::::::::::::::::::: challenge

What is the solution to this exercise?

::::::::::::::::::::::::::::::::::::::::::::: solution

This is the solution to this exercise.

::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

:::::::::::::::::::::::::::::::::::::::

````

::::::::::::::::::::::::::::::::::::::: challenge

What is the solution to this exercise?

::::::::::::::::::::::::::::::::::::::::::::: solution

This is the solution to this exercise.

::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

:::::::::::::::::::::::::::::::::::::::

```

Reply:
😆 ok looks like GitHub's suggestion feature does not play nicely with Markdown code blocks.

Reply:
I had to make this suggestion as a PR to your branch instead: https://github.com/carpentries/maintainer-onboarding/pull/74